num_of_buttons: 13 
axis_tilt: 1
axis_pan: 0
axis_linear_z: 5
scale_linear_z: -0.2 
scale_pan: 1.0
scale_tilt: -3.1416
button_speed_up: 4
button_speed_down: 2
button_dead_man_torso: 7
cmd_topic:  /joint_commands
